source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt

```diff
@@ -5222,8 +5222,10 @@ Octopus.Client.Model
   class XOptionsResource
   {
     static System.String XFrameAllowFromDescription
+    static System.String XFrameOptionsDescription

```

Comment:
Do these need to be public? Seems... odd.

Reply:
yes does not make sense to me either.. i will remove it

Reply:
I think it was done this way because XOptionsResource on server also has the descriptions as public